### ISSUE TITLE
"no-extra-semicolons" rule not properly linting semicolons after css @apply

### ISSUE THREAD

**Issue by iamfrntdv** [stylelint/stylelint#2402]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

"no-extra-semicolons" rule throws error if after css apply comes comment. According to documentation there should be semicolon after every mixin declaration. But if after mixin comes comment stylelint throws error. 

> Which rule, if any, is this issue related to?

"no-extra-semicolons"

> What CSS is needed to reproduce this issue?

```css
--some-mixin{
   color:red;
};

/* Some Comment */
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "no-extra-semicolons": true
  }
}
```

> Which version of stylelint are you using?

 `7.9.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

as a part of gulp task

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No.

> What did you expect to happen?

no error and no warning

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
test.css
1:21    !! Unexpected extra semicolon (no-extra-semicolons)
```

**Comment by jeddy3** [comment]
@iamfrntdv Thanks for reporting and for using the template. This has already been reported in https://github.com/stylelint/stylelint/issues/2395. That issue is labelled as `help wanted`, and so feel free to contribute to the fix.
